snap/snapcraft.yaml

```diff
@@ -0,0 +1,126 @@
+%YAML 1.1
+---
+# app specific top-level metadata
+name: mazu-editor
+title: Mazu Editor
+summary: A minimalist text editor with syntax highlight, copy/paste, and search
+description: |
+    **Usage**
+
+    Command line: (_filename_ is optional)
+
+        me _filename_
+
+    Supported keys:
+
+    * Ctrl-S: Save
+    * Ctrl-Q: Quit
+    * Ctrl-F: Find string in file
+        - ESC to cancel search, Enter to exit search, arrows to navigate
+    * Ctrl-C: Copy line
+    * Ctrl-X: Cut line
+    * Ctrl-V: Paste line
+    * PageUp, PageDown: Scroll up/down
+    * Up/Down/Left/Right: Move cursor
+    * Home/End: move cursor to the beginnging/end of editing line
+
+    Mazu Editor does not depend on external library (not even curses). It uses fairly
+    standard VT100 (and similar terminals) escape sequences.
+
+    **Acknowledge**
+
+    Mazu Editor was inspired by excellent tutorial Build Your Own Text Editor:
+
+    https://viewsourcecode.org/snaptoken/kilo
+
+    **Snap-specific information**
+
+    Due to the confined nature of the snap apps, by default this snap can only access
+    the files under your home directory(excluding the dotfiles right under it).
+
+    To open and save files under the `/mnt` and the `/media` directories, connect the
+     snap to the `removable-media` interface by running the following command in a
+     terminal:
+
+        sudo snap connect mazu-editor:removable-media
+
+    This is NOT an official distribution of Mazu Editor, for any problems regarding
+    the use of this snap please refer to the snap's issue tracker:
+
+    https://github.com/Lin-Buo-Ren/mazu-editor-snap/issues
```

Comment:
Change this URL back to the original GitHub repository.

Reply:
Oops.  I've dropped the entire section, please review